src/node/desktop/src/core/file-path.ts

```diff
@@ -35,10 +35,10 @@ const homePathAlias = '~/';
 const homePathLeafAlias = '~';
 
 function normalizeSeparators(path: string, separator = '/') {
-  return path.replace(/[\\/]/g, separator);
+  return path.replace(/[\\/]+/g, separator);
 }
 
-function normalizeSeparatorsNative(path: string) {
+export function normalizeSeparatorsNative(path: string) {
   const separator = process.platform === 'win32' ? '\\' : '/';
```

Comment:
Use Node's constant for path separators. `path.sep`

https://nodejs.org/dist/latest-v16.x/docs/api/path.html#pathsep

Reply:
Alright